appserver/transaction/jts/src/main/java/com/sun/jts/CosTransactions/RecoveryManager.java

```diff
@@ -58,6 +52,12 @@
 import org.omg.CORBA.SystemException;
 import org.omg.CORBA.TRANSIENT;
 import org.omg.CosTransactions.Status;
+
+import com.sun.enterprise.transaction.jts.api.TransactionRecoveryFence;
```

Comment:
Hmmm, I would keep simple aplhabetical ordering in imports, so we would not fight in each part again and again (here I am not a part of the fight)  :dagger:

Reply:
It's the auto-ordering of the IDE ;)